content/kubermatic/main/architecture/requirements/cluster-requirements/_index.en.md

```diff
@@ -50,27 +50,29 @@ If you have more than one network adapter, and your Kubernetes components are no
 
 ## Check Required Ports
 
-The tables below list the ports that need to be open for communication, e.g. by allowing them via a firewall.
-
-### Master Cluster Master Node(s)
-
-| Protocol | Direction | Port Range | Purpose                 |
-|----------|-----------|------------|-------------------------|
-| TCP      | Inbound   | 6443*      | Kubernetes API server   |
-| TCP      | Inbound   | 2379-2380  | etcd server client API  |
-| TCP      | Inbound   | 10250      | kubelet API             |
-| TCP      | Inbound   | 10251      | kube-scheduler          |
-| TCP      | Inbound   | 10252      | kube-controller-manager |
-| TCP      | Inbound   | 10255      | Read-only kubelet API   |
-
-### Worker Node(s) & User Cluster Worker Nodes
-
-| Protocol | Direction | Port Range  | Purpose               |
-|----------|-----------|-------------|-----------------------|
-| TCP      | Inbound   | 10250       | kubelet API           |
-| TCP      | Inbound   | 10255       | Read-only kubelet API |
-| TCP      | Inbound   | 30000-32767 | NodePort Services**   |
-
-** Default port range for [NodePort Services](https://kubernetes.io/docs/concepts/services-networking/service/).
+Please ensure that all nodes in a user cluster can communicate without restriction to ensure functionality of CNI/CSI and Kubernetes itself.
+In addition, user cluster nodes must be able to connect to the Seed cluster's nodeport-proxy. This depends on the [expose strategy]({{< ref "../../../tutorials-howtos/networking/expose-strategies" >}}.
+
+It is recommended to make yourself familiar with the [concept of networking]({{< ref "../../../architecture/concept/kkp-concepts/networking" >}}) in KKP.
+
+| Source                  | Destination                   | Expose Strategy | Ports             | Purpose                                              |
+|-------------------------|-------------------------------|-----------------|-------------------|------------------------------------------------------|
+| KKP Users               | Master Ingress Controller     | Any             | 443*              | Access to KKP Dashboard                              |
+| KKP Operator            | Master cluster Kubernetes API | Any             | 6443*             | Operator access                                      |
```

Comment:
I suspect we can drop this since the operator will always be deployed into the master cluster, no?